metrics/metrics.go

```diff
@@ -97,6 +97,7 @@ var (
 	TiKVUnsafeDestroyRangeFailuresCounterVec *prometheus.CounterVec
 	TiKVPrewriteAssertionUsageCounter        *prometheus.CounterVec
 	TiKVGrpcConnectionState                  *prometheus.GaugeVec
+	TiKVAggressiveLockedKeysCounter          *prometheus.CounterVec
```

Comment:
Do we also need the metric that we could calculate:
- The `aggressive locking keys /  The total locking keys`, so we know how many keys are affected by the locking mode.
- The `aggressive locked transactions/ The total pessimistic transactions`, so we know the percentage affected from transaction perspective.

Reply:
Oh I see them in the tidb pr.